src/main/java/racingcar/domain/Car.java

```diff
@@ -4,22 +4,44 @@
 
 import java.util.Objects;
 
+import static racingcar.exception.Message.MAX_LENGTH_ERROR;
+import static racingcar.exception.Message.NULL_OR_EMPTY_ERROR;
+
 public class Car {
     private int position;
+    private String name;
+    private final int LIMIT = 5;
```

Comment:
```suggestion
    private static final int LIMIT = 5;
```

변경되지 않을 상수라면 static 을 선언해도 좋을 듯 합니다.